src/core/core.ts

```diff
@@ -74,7 +77,7 @@ class Core {
    * Loop through the routes and set them
    */
   private setRoutes(): void {
-    ['model', 'collection'].forEach(route => this.setRoute(route));
+    ['model', 'collection'].forEach((route: Route) => this.setRoute(route));
     // [Route.MODEL, Route.COLLECTION].forEach(route => this.setRoute(route));
```

Comment:
So, this line is apparently the right one for me. It works as far as showing no errors, but when I compile it fails :D

Reply:
I wasn't able to get the typing on it with the shorthand so I wrapped it up. Hmm... odd, I wonder if we have a version discrepancy between something we're running.